### PR TITLE
fix deposit calculation in account widget

### PR DESCRIPTION
**Pull request by punkrussian72**

### CODE REVIEW

File: src/components/Header/WalletManager/WalletManager.tsx [hunk 1]
```diff
@@ -14,6 +14,8 @@ import GetKSMModal from 'components/GetKSMModal/GetKSMModal';
 import { formatKusamaBalance } from 'utils/textUtils';
 import BalanceSkeleton from '../../Skeleton/BalanceSkeleton';
 import config from '../../../config';
+import { BN } from '@polkadot/util';
```
Comment (SergeyOkishev) [inline src/components/Header/WalletManager/WalletManager.tsx:17]: please sort imports:
1. Libs
2. "@smth"
3. aliases
4. relative paths.

File: src/components/Header/WalletManager/WalletManager.tsx [hunk 2]
```diff
@@ -76,11 +78,22 @@ export const WalletManager: FC = () => {
...
-    if (!selectedAccount?.deposits?.sponsorshipFee || selectedAccount?.deposits?.sponsorshipFee?.toString() === '0') {
+    if (!selectedAccount?.deposits) {
       return undefined;
     }
-    return formatKusamaBalance(selectedAccount?.deposits?.sponsorshipFee?.toString());
-      }, [selectedAccount?.deposits]);
+
+    const { sponsorshipFee, bids } = selectedAccount.deposits;
+    const { leader, withdraw } = bids || { leader: [], withdraw: [] };
+    if ((!sponsorshipFee?.isZero() || leader.length !== 0 || withdraw.length !== 0)) {
```
Comment (SergeyOkishev) [inline src/components/Header/WalletManager/WalletManager.tsx:87]: if ((sponsorshipFee?.isZero() || leader.length === 0 || withdraw.length === 0)) { return undefined }

less nesting - cleaner code

**Review by SergeyOkishev — CHANGES_REQUESTED** [review]

File: src/components/Header/WalletManager/WalletManager.tsx (hunk 1 above)
Comment (punkrussian72) [reply src/components/Header/WalletManager/WalletManager.tsx]: fixed

File: src/components/Header/WalletManager/WalletManager.tsx (hunk 2 above)
Comment (punkrussian72) [reply src/components/Header/WalletManager/WalletManager.tsx]: fixed

**Review by SergeyOkishev — APPROVED** [review]

**Pull request merged**